public/js/util/configureStore.js

```diff
@@ -19,5 +11,14 @@ const createStoreWithMiddleware = compose(
 )(createStore);
 
 export default function configureStore(initialState) {
-  return createStoreWithMiddleware(rootReducer, initialState);
+
+  const store = createStoreWithMiddleware(rootReducer, initialState);
+
+  if (module.hot) {
```

Comment:
This allows us to trigger differently depending on what's changed... here we trigger a reduce replacement (rather than a re-render)